server/grpc/api/service.go

```diff
@@ -22,8 +22,18 @@ func NewServiceServer(sdk *sdk.SDK) *ServiceServer {
 }
 
 // Create creates a new service from definition.
-func (s *ServiceServer) Create(ctx context.Context, request *protobuf_api.CreateServiceRequest) (*protobuf_api.CreateServiceResponse, error) {
-	definition := fromProtoService(request.Definition)
+func (s *ServiceServer) Create(ctx context.Context, req *protobuf_api.CreateServiceRequest) (*protobuf_api.CreateServiceResponse, error) {
+	definition := fromProtoService(&types.Service{
```

Comment:
Why not directly updating `fromProtoService` to accept the struct `CreateServiceResponse`?
`fromProtoService` is only used here. You could also renamed it to something more explicit.

Reply:
Actually, what did we say about sdk also? To creates a new struct if there are too many parameters, no?
`CreateServiceParameters`?

Reply:
I feel like passing one struct is easier here. I understand why api requirement was to pass exact data that create (or any other method) expect, but internally `Service` struct could be used. 

We can merge this to finish stabilize api and think how we want to approach this pbm.

Reply:
> We can merge this to finish stabilize api and think how we want to approach this pbm.
👍

Reply:
Issue created: https://github.com/mesg-foundation/engine/issues/1147